src/Apps/Ingress/EntryPoints/IngressApp/IngressApp.types.ts

```diff
@@ -0,0 +1,28 @@
+import { IStyle, IStyleFunctionOrObject } from '@fluentui/react';
+import { MockAdapter } from '../../../../Adapters';
+import IngressAdapter from '../../../../Adapters/IngressAdapter';
+import { ICardBaseProps } from '../../../../Models/Constants';
+import { IExtendedTheme } from '../../../../Theming/Theme.types';
+
+export interface IIngressAppProps extends ICardBaseProps {
```

Comment:
do we need to extend from ICardBaseProps?

Reply:
good call, I'll remove this too